package-lock.json

```diff
@@ -0,0 +1,4737 @@
+{
```

Comment:
let's not include the package-lock.json just on commits. Do it once we finalize all dependencies